### PR TITLE
1.7.0 version bump

### LINKED ISSUE #14: Bump "Tested up to" to 4.7
Couldn't find any issues so far. There are only a few cosmetic bugs: https://github.com/wearerequired/WP-Widget-Disable/milestone/2?closed=1

- [ ] Update changelog
- [ ] Update meta data
- [ ] Bump versions

### PR DESCRIPTION
**Pull request by swissspidy**
* Version bump
* Changelog update

Fixes #14.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,17 @@
 # Changelog
 
+## 1.5.0
```
Comment (ocean90) [inline CHANGELOG.md]: The changelog should contain all versions IMO.

Comment (swissspidy) [inline CHANGELOG.md]: Having to translate years old changelog entries is a burden for translators though. Also, the readme gets very long. See FAQ: https://developer.wordpress.org/plugins/wordpress-org/plugin-developer-faq/#what-should-be-in-my-changelog

Comment (ocean90) [inline CHANGELOG.md]: That's right, but this is CHANGELOG.md, not README.md.

Comment (swissspidy) [inline CHANGELOG.md]: Gotcha. Adding 1.7.0 and 1.6.0 to CHANGELOG.md makes sense.

**Pull request merged**